event-stream-processing/src/node-css-parser.ts

```diff
@@ -0,0 +1,40 @@
+import { injectable } from 'inversify';
+import { Parser } from './parser.isvc';
+import * as lodash from 'lodash';
+import { renameField } from './shared/rename-field';
+
+@injectable()
+export class NodeCssParser implements Parser {
```

Comment:
Add a [jsdoc ](https://jsdoc.app/) header, explaining a bit about the use case/purpose (you can use a long and a [summary ](https://jsdoc.app/tags-summary.html) description for that) of this class. Make sure to include the showcase team as an author, as well as yourself.